sale_order_import/wizard/import_sale.py

```diff
@@ -521,7 +521,10 @@ def import_sale_data(self):
             quotechar='"',
             delimiter=','
         )
-        sheet_fields = next(csv_iterator)
+        try:
+            sheet_fields = next(csv_iterator)
+        except:
+            raise Warning(_("Please import a CSV file with UTF-8 encoding."))
```

Comment:
What about this method? https://github.com/TimLai125/sts-custom/blob/d34f3e681aa999c60b1dbf2642ab9f14632f63c6/sale_order_import/wizard/import_sale.py#L419-L428

Reply:
This method is used in v8, I have deleted the method in the next commit.